mkdocs.yml

```diff
@@ -171,19 +171,39 @@ plugins:
       # FIX ME: Add autodetection to translation rates so that only languages
       # with >30% translations are built. See https://www.transifex.com/opengisch/qfield-documentation/dashboard/
       languages:
-        en: English
-        de: Deutsch
-        fr: Français
-        it: Italiano
+        en:
+          name: English
+          site_name: QField Ecosystem Documentation
+        de:
+          name: Deutsch
+          site_name: QField Ecosystem Documentation # TRANSLATE ME
+        fr:
+          name: Français
+          site_name: Documentation de l'écosystème QField
+        it:
+          name: Italiano
+          site_name: QField Ecosystem Documentation # TRANSLATE ME
         # rm: Romansh
-        ja: 日本語
+        ja:
+          name: 日本語
+          site_name: QField Ecosystem Documentation # TRANSLATE ME
         # gl: Galician
-        pt: Portuguese
-        es: Español
+        pt:
+          name: Portuguese
+          site_name: QField Ecosystem Documentation # TRANSLATE ME
```

Comment:
@jonnyforestGIS can you help us?

Reply:
I will try today at night. Cheers

Reply:
@m-kuhn the translation is: Documentação do Ecossitema QField

Reply:
@why-not-try-calmer I realize now that it's possible to put the site_description in this commit sorry to not put it when I add the site_name. If you have time I add in here the Portuguese site_description:


Bem-vindo à documentação do ecossistema QField - um conjunto de produtos projetados para tornar o trabalho de campo contínuo e eficiente. Nestas páginas, fornecemos documentação abrangente e fácil de entender para todos os nossos produtos. Caso seja um novo utilizador ou um profissional experiente, encontrará tudo o que precisa de saber sobre o uso do QField e as suas respetivas ferramentas.

Cheers